libraries/AP_RPM/AP_RPM.cpp

```diff
@@ -199,6 +200,16 @@ void AP_RPM::update(void)
             }
 
             drivers[i]->update();
+
+#if HAL_WITH_ESC_TELEM
+            if (_params[i].type != RPM_TYPE_ESC_TELEM && _params[i].esc_mask > 0 && healthy(i)) {
```

Comment:
shouldn't this be == RPM_TYPE_ESC_TELEM

Reply:
no, actually. A comment would be useful there. This is functioning as the opposite of RPM_TYPE_ESC_TELEM

RPM_TYPE_ESC_TELEM = read ESC_TELEM and store it in AP_RPM
RPM_TYPE_EverythingElse = read RPM and write to ESC_TELEM

Reply:
@andyp1per I've added a comment.